code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2521,7 +2516,7 @@
 					for(var/datum/disease/D in M.viruses)
 						D.spread = "Remissive"
 						D.stage--
-						if(D.stage < 1)
+						if(D.stage >= 1)
```

Comment:
I'm not sure if this makes sense? It's supposed to keep decreasing stage by 1 until its below 1 in which case it cures it